hudi-client/src/main/java/org/apache/hudi/config/HoodieMetricsConfig.java

```diff
@@ -58,6 +59,12 @@
 
   public static final String GRAPHITE_METRIC_PREFIX = GRAPHITE_PREFIX + ".metric.prefix";
 
+  // User defined
+  public static final String USER_DEFINED_REPORTER_PREFIX = METRIC_PREFIX + ".user.defined";
+  public static final String USER_DEFINED_REPORTER_CLASS = USER_DEFINED_REPORTER_PREFIX + ".class";
```

Comment:
would rename to METRICS_REPORTER_CLASS = "metrics.reporter.class" ?

Reply:
the key name will be like "hoodie.metrics.reporter.class" ?